pkg/kubectl/cmd/get/get_test.go

```diff
@@ -813,7 +839,7 @@ func TestGetListComponentStatus(t *testing.T) {
 	cmd.Run(cmd, []string{"componentstatuses"})
 
 	expected := `NAME            STATUS      MESSAGE   ERROR
-servergood      Healthy     ok        
```

Comment:
looks like bazel-test is failing because of these whitespace changes.
other than that LGTM.

Reply:
Good catch, lemme remove that.